Makefile

```diff
@@ -44,3 +59,34 @@ format :
 	# show diff via black
 	black tests --diff
 	black stream2hop --diff
+
+print-%  : ; @echo $* = $($*)
+
+
+.PHONY: tns_container
+tns_container: 
+	@if [ ! -z "$$(git status --porcelain)" ]; then echo "Directory is not clean. Commit your changes."; exit 1; fi
+	docker build -f Dockerfile_TNS -t $(CNT_IMG) .
+	docker tag $(CNT_IMG) $(CNT_LTST)
+
+.PHONY: set-release-tags
+set-release-tags:
+	@$(eval RELEASE_TAG := $(shell echo $(GITHUB_REF) | awk -F- '{print $$2}'))
+	@echo RELEASE_TAG =  $(RELEASE_TAG)
+	@$(eval MAJOR_TAG   := $(shell echo $(RELEASE_TAG) | awk -F. '{print $$1}'))
+	@echo MAJOR_TAG = $(MAJOR_TAG)
+	@$(eval MINOR_TAG   := $(shell echo $(RELEASE_TAG) | awk -F. '{print $$2}'))
+	@echo MINOR_TAG = $(MINOR_TAG)
+
+.PHONY: docker_push
+docker_push: 
+	./scripts/awsDockerLogin $(REGION) $(AWSREG) >/dev/null 2>/dev/null
```

Comment:
Actually, looks like it's named `./scripts/awsDockerLogin.sh`. The error is probably getting hidden by the `2>/dev/null`, which you might want to remove.

Reply:
will do